js/runner.go

```diff
@@ -319,7 +319,8 @@ func (r *Runner) runPart(ctx context.Context, out chan<- stats.SampleContainer,
 		return goja.Undefined(), err
 	}
 
-	v, _, err := vu.runFn(ctx, group, fn, vu.Runtime.ToValue(arg))
+	//nolint: dogsled
+	v, _, _, _, err := vu.runFn(ctx, group, fn, vu.Runtime.ToValue(arg))
```

Comment:
You can drop the `lib.State` from the returns as it's always not used and remove the `nolint`. Maybe some git archaeology will show you why it was added and when it was removed ;)

Reply:
@MStoykov what do you mean? `v` is used and is returned in lines below. Here `dogsled` warns about too many blank assignment.

Reply:
the `v` is used but the second returns is a `lib.State` that is never used anywhere so we can just not return it ;)

Reply:
@MStoykov ah ok, good to know. I will do it in other PR, for easier to review, revert if anything broken. Is it ok?

Reply:
you are literally touching all the three lines that will be changed in this PR, so I would prefer if it's done here instead of having to look through another PR

Reply:
ok